src/fsharp/FSComp.txt

```diff
@@ -1551,6 +1552,8 @@ forFormatInvalidForInterpolated4,"Interpolated strings used as type IFormattable
 3384,scriptSdkNotDeterminedUnexpected,"The .NET SDK for this script could not be determined. If the script is in a directory using a 'global.json' then ensure the relevant .NET SDK is installed. Unexpected error '%s'."
 3384,scriptSdkNotDeterminedNoHost,"The .NET SDK for this script could not be determined. dotnet.exe could not be found ensure a .NET SDK is installed."
 3385,tcInvalidStructReturn,"The use of '[<Struct>]' on values, functions and methods is only allowed on partial active pattern definitions"
+3386,tcImplicitConversionUsed,"This expression uses an implicit conversion to convert type '%s' to type '%s'. Warnings are enabled for implicit conversions. Consider using an explicit conversion or disabling this warning."
+3387,tcAmbiguousImplicitConversion,"This expression has type '%s' and is only made compatible with type '%s' through an ambiguous implicit conversion. Consider using an explicit call to 'op_Implicit'."
```

Comment:
And for this ambiguous overload case, doing the now-common full list of potential members would be very useful.  These lists are used in FSAC to power the quick fixes and I at least use them often!

Reply:
Yes agreed, thanks!  I'll make that change